src/pages/variantHistory/ColumnHeaders.tsx

```diff
@@ -30,7 +31,9 @@ const ColumnHeaders: React.FC<ColumnHeadersProps> = ({ columns, loading }) => {
       })}
       {loading &&
         Array.from(Array(columnLimit)).map((i) => (
-          <HeaderCell key={`loading_cell_${i}`}>Loading...</HeaderCell>
+          <HeaderCell key={`loading_cell_${i}`}>
```

Comment:
Would the `i` here have value `undefined`? That's what I'm seeing when I test it out

Reply:
Thanks fixed!